src/components/item/sharing/ItemSharingTab.js

```diff
@@ -66,23 +65,15 @@ const ItemSharingTab = ({ item, memberships }) => {
       return null;
     }
 
-    const membershipsWithoutSelf = membershipsWithoutUser(
-      memberships,
-      currentMember.get('id'),
-    );
-    const authorizedMemberships = membershipsWithoutSelf?.filter(
-      ({ memberId }) => {
-        const member = members?.find(({ id: mId }) => mId === memberId);
-        return !member?.email?.includes(PSEUDONIMIZED_USER_MAIL);
-      },
-    );
+    const authorizedMemberships = memberships?.filter(({ memberId }) => {
+      const member = members?.find(({ id: mId }) => mId === memberId);
+      return !member?.email?.includes(PSEUDONIMIZED_USER_MAIL);
+    });
 
-    const authenticatedMemberships = membershipsWithoutSelf?.filter(
-      ({ memberId }) => {
-        const member = members?.find(({ id: mId }) => mId === memberId);
-        return member?.email?.includes(PSEUDONIMIZED_USER_MAIL);
-      },
-    );
+    const authenticatedMemberships = memberships?.filter(({ memberId }) => {
```

Comment:
Nitpick: could use a partitioning function for a single traversal since the filter is now applied twice to obtain two collections with the exact opposite predicate, i.e. `const [authorizedMemberships, authenticatedMemberships] = partition(memberships, ({ memberId }) => ...)` e.g. https://stackoverflow.com/questions/11731072/dividing-an-array-by-filter-function